src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/RoatpGatewayPageViewModel.cs

```diff
@@ -13,27 +13,19 @@ public class RoatpGatewayPageViewModel
         public string Status { get; set; }
 
         public string Ukprn { get; set; }
-        public string UkrlpLegalName { get; set; }
-
-        // It will not be needed any more. It was for the breadcrumb, but the logic for it changed
-        public string GatewayReviewStatus { get; set; }
+        public string ApplyLegalName { get; set; }
 
-        // these two will disappear once views use them locally - retained to keep Greg's UX stuff
-         public List<TabularData> Tables { get; set; }   
-         //the SummaryList items are only here to preserve the UX stuff Greg did, and these will disappear once we've done the stories with 1+ tables (section 2)
-        public TabularData SummaryList { get; set; }  // this will disappear once section 2 work starts
-     
         public string OptionPassText { get; set; }
         public string OptionFailText { get; set; }
-
         public string OptionInProgressText { get; set; }
         public List<ValidationErrorDetail> ErrorMessages { get; set; }
 
-        public string ApplyLegalName { get; set; }
+   
         public DateTime? SourcesCheckedOn { get; set; }
         public DateTime? ApplicationSubmittedOn { get; set; }
 
         public string Heading { get; set; }
         public string Caption { get; set; }
+        public string NoSelectionErrorMessage { get; set; }
```

Comment:
Should be marked as virtual so that the inherited class can override it.

Reply:
Why we need that? I haven't come across such cases, when it needs to be overridden.  I would suggest to be changed only when there is a need for it.